cmd/kubelet/app/options/options.go

```diff
@@ -212,6 +212,9 @@ func NewKubeletConfiguration() (*kubeletconfig.KubeletConfiguration, error) {
 		return nil, err
 	}
 	applyLegacyDefaults(config)
+	if err := config.Mutate(); err != nil {
```

Comment:
@haircommander @sohankunkerkar just checking, this will not interfere with the new drop-in configuration feature? 

I don't think there are any reasons to do the mutation AFTER all the merges. If customer applied a bad config, it is their responsibility. Just checking in case you may think of a problem here